packages/shared/types.ts

```diff
@@ -392,6 +392,15 @@ export type AvatarAsset = {
   }>
 }
 
+export type Notification = {
+  type: number
+  message: string
+  buttonMessage: string
+  timer: number
```

Comment:
Adding a comment to know the unit of the timer:
```suggestion
  timer: number // in seconds
```